### ISSUE TITLE
Skin resetting issue (SPMC 16.4.2) on Android MX2

### ISSUE THREAD

**Issue by nicgentile** [koying/SPMC#439]
Hi,

The skin keeps resetting no matter what I do, randomly. Have modified settings.xml to reflect what I want, and that doesn't do much, use Script Manager to automatically reset guisettings.xml, settings.xml and it still ignores, and rewrites to skin.re-touched randomly, when there is a power outage, ignores the guisettings.xml I have created, or the backup, or settings.xml, or its backup and defaults back to skin.re-touched.

Logs show nothing, and I reproduce the guisettings.xml file from the one Kodi produces when i first put the settings as I want them. I have reproduced this bug on 8 devices so far.

While I get that guisettings.xml must be written and rewritten every so often, isn't there a way to stop this? Setting the file to read-only doesn't resolve the issue either.

Thanks.

**Comment by koying** [comment]
You know that skin settings are now saved in their own subdir of userdata/addon_data ?

**Comment by nicgentile** [comment]
Yes, that I know, and even have a backup of that, and have it copied in at every reboot, since it too randomly sets to default, and, guisettings.xml keeps resetting randomly every so often as well. defaulting back to the original theme. If I set it to the theme I want, then it sticks. The only solution I have so far is to remove everything in the skin.re-touched folder, put my theme there, then modify addon.xml so that when this happens, the default holds.

For instance, the other day, we had a 2 minute internet outage, and the box froze, and when we rebooted it after a 4 minute wait, it had reset itself back to default. Script Manager overwrote the file at reboot as I have set it, but, SPMC also rewrote over what Script Manager have written, which was a perfectly written guisettings.xml file, in fact, a backup of what SPMC originally created.
